src/probability/dirac.jl

```diff
@@ -2,21 +2,17 @@
 export Dirac
 
 struct Dirac{X} <: AbstractMeasure
-    supp :: X
```

Comment:
Rename to `x` as `supp` seems to be a set, not a value.

Reply:
Wouldn't `Dirac(1.0)` make sense? OTOH we may sometimes want this to be represented as `::Dirac{Float64}`, or possibly (for static analysis) as `::Dirac{Val{1.0}}`. But the first seems simpler to start with.

Reply:
Then again, the value determines a singleton set, which _is_ the support. And maybe `supp` would carry over more uniformly to other measures? So, ok maybe I've convinced myself I agree